suite-native/navigation/src/navigators.ts

```diff
@@ -1,4 +1,5 @@
 import { NavigatorScreenParams } from '@react-navigation/native';
+import { ExchangeTrade } from 'invity-api';
```

Comment:
This should fix the depcheck 🤞 

```suggestion
import type { ExchangeTrade } from 'invity-api';
```

Reply:
Fixed: 86dd433264d8ece4a60eafd8c9839e9d9cad1b71